### ISSUE TITLE
[Error] When building: Cannot find name 'IObliqueParams'.

### ISSUE THREAD

**Issue by pjoe** [brianzinn/react-babylonjs#300]
When trying to build I get this error, could be related to [this change](https://github.com/BabylonJS/Babylon.js/commit/6c81ca30f4bf2b61dd84cdff4c9f42bfe1f43937#diff-52b1f4a8c053e15cede45c07d22676ddc9b9c5190491c87da782115926be6272R28)?
```
> npm run build

> react-babylonjs@3.1.20 prebuild
> rimraf dist


> react-babylonjs@3.1.20 build
> cross-env NODE_ENV=production rollup -c

ENV: production react-babylonjs

src/react-babylonjs.ts → dist/react-babylonjs.js...
(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
@babylonjs/core/Engines/engine.js (imported by src/Engine.tsx)
@babylonjs/core/Misc/observable.js (imported by src/Engine.tsx)
react-reconciler (imported by src/render.ts)
@babylonjs/core/Events/pointerEvents.js (imported by src/Scene.tsx)
@babylonjs/core/Physics/physicsEngineComponent.js (imported by src/Scene.tsx)
@babylonjs/core/scene.js (imported by src/Scene.tsx)
[!] (plugin rpt2) Error: .../react-babylonjs/packages/react-babylonjs/src/generatedProps.ts(831,19): semantic error TS2304: Cannot find name 'IObliqueParams'.
src/generatedProps.ts
Error: .../react-babylonjs/packages/react-babylonjs/src/generatedProps.ts(831,19): semantic error TS2304: Cannot find name 'IObliqueParams'.
    at error (...\react-babylonjs\node_modules\rollup\dist\shared\rollup.js:198:30)     
    at throwPluginError (...\react-babylonjs\node_modules\rollup\dist\shared\rollup.js:21718:12)
    at Object.error (...\react-babylonjs\node_modules\rollup\dist\shared\rollup.js:22672:20)
    at Object.error (...\react-babylonjs\node_modules\rollup\dist\shared\rollup.js:21895:42)
    at RollupContext.error (...\react-babylonjs\packages\react-babylonjs\node_modules\rollup-plugin-typescript2\src\rollupcontext.ts:37:18)
    at ...\react-babylonjs\packages\react-babylonjs\node_modules\rollup-plugin-typescript2\src\print-diagnostics.ts:41:11
    at arrayEach (...\react-babylonjs\packages\react-babylonjs\node_modules\rollup-plugin-typescript2\node_modules\lodash\lodash.js:530:11)
    at Function._.each [as forEach] (...\react-babylonjs\packages\react-babylonjs\node_modules\rollup-plugin-typescript2\node_modules\lodash\lodash.js:9410:14)
    at printDiagnostics (...\react-babylonjs\packages\react-babylonjs\node_modules\rollup-plugin-typescript2\src\print-diagnostics.ts:9:2)
    at ...\react-babylonjs\packages\react-babylonjs\node_modules\rollup-plugin-typescript2\src\index.ts:216:6
```

**Comment by pjoe** [comment]
FWIW: pinning BJS to `~6.24.0` seems to get around this.

**Comment by brianzinn** [comment]
I have a PR that was just merged in @babylonjs/core.  For now I put "any" in the generated code where it put `IObliqueParams`.

**Comment by brianzinn** [comment]
It won't be fixed unti they release another version.

**Comment by pjoe** [comment]
This is fixed with BJS 6.26.0, closing
